tests/e2e/MediaCo/portal.spec.js

```diff
@@ -108,8 +110,23 @@ test.describe('E2E test', () => {
     const sendToMgr = page.locator('mat-checkbox[data-test-id="C3B43E79AEC2D689F0CF97BD6AFB7DC4"]');
     await sendToMgr.click();
 
+    const currentCaseID = await page.locator('div[id="current-caseID"]').textContent();
+    const filePath = path.join(__dirname, '../../../src/assets/cableinfo.png');
+    await page.setInputFiles('#upload-photo', filePath);
```

Comment:
Since this can be used for uploads in general, perhaps we should change the name to something more generic such as upload-input?